hydra-module.nix

```diff
@@ -64,7 +67,7 @@ with rec {
       };
 
       package = mkOption {
-        type = types.path;
+        type = types.package;
```

Comment:
The NixOS module documentation recommends `types.package` over `types.path`, but there may be some people taking advantage of the current state of affairs; feel free to let me know if this is the case.